src/biogeochem/ch4Mod.F90

```diff
@@ -2174,22 +2236,22 @@ subroutine ch4 (bounds, num_soilc, filter_soilc, num_lakec, filter_lakec, &
          do fc = 1, num_lakec
             c = filter_lakec(fc)
 
-            if (.not. ch4_first_time(c)) then
+            if (.not. ch4_first_time_col(c)) then
```

Comment:
Note that my four tests PASS without a similar if statement in the gridcell-level check (next do loop).

@billsacks I feel this is ready for review and then I will run the full test suite.

Reply:
From your wording I understand that I may not need both the col and grc level variables if I have the grc level variable. That doesn't seem right to me (though I could easily be wrong), so I made a grc level copy of the col level variable for use in the gridcell level balance check.

Reply:
Oh, now I caught what you meant by "subgrid level is arbitrary."